FusionIIIT/templates/iwdModuleV2/viewWork.html

```diff
@@ -1,6 +1,6 @@
 <div class="fields">
     <div class="eight wide field">
-        <form class="ui form" style="padding: 8px; padding-left: 24px; padding-right: 24px;", method="post" action="http://172.27.16.216/iwdModuleV2/page1View/">{% csrf_token %}
+        <form class="ui form" style="padding: 8px; padding-left: 24px; padding-right: 24px;", method="post" action="page1View">{% csrf_token %}
```

Comment:
the page is not being redirected to page1View

Reply:
Here our task was just to make end point and as create view requisition was not showing anything I do this.
For proper redirect to page1View that was this weeks task I have generated a PR Today
https://github.com/FusionIIIT/Fusion/pull/1073